manager/testdata/notebook/read-flow-tls/setup-certs.sh

```diff
@@ -2,12 +2,13 @@
 # Copyright 2021 IBM Corp.
 # SPDX-License-Identifier: Apache-2.0
 
+kubectl create ns fybrik-system
```

Comment:
should we define `fybrik-system` as a variable ? it is returned to itself a lot of times

Reply:
I fixed that Thanks